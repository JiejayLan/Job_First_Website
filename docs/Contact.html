<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>JSDoc: Class: Contact</title>

    <script src="scripts/prettify/prettify.js"> </script>
    <script src="scripts/prettify/lang-css.js"> </script>
    <!--[if lt IE 9]>
      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <link type="text/css" rel="stylesheet" href="styles/prettify-tomorrow.css">
    <link type="text/css" rel="stylesheet" href="styles/jsdoc-default.css">
</head>

<body>

<div id="main">

    <h1 class="page-title">Class: Contact</h1>

    




<section>

<header>
    
        <h2><span class="attribs"><span class="type-signature"></span></span>Contact<span class="signature">()</span><span class="type-signature"></span></h2>
        
            <div class="class-description"><p>The class Contact will display the website's contact page, where
anyone can learn how to contact the site managers for support</p></div>
        
    
</header>

<article>
    <div class="container-overview">
    
        

    
    <h2>Constructor</h2>
    

    
    <h4 class="name" id="Contact"><span class="type-signature"></span>new Contact<span class="signature">()</span><span class="type-signature"></span></h4>
    

    















<dl class="details">

    

    

    

    

    

    

    

    

    

    

    

    

    
    <dt class="tag-source">Source:</dt>
    <dd class="tag-source"><ul class="dummy"><li>
        <a href="pages_contact.js.html">pages/contact.js</a>, <a href="pages_contact.js.html#line14">line 14</a>
    </li></ul></dd>
    

    

    

    
</dl>




















    
    </div>

    

    

    

    

    

    

    

    

    

    
</article>

</section>




</div>

<nav>
    <h2><a href="index.html">Home</a></h2><h3>Classes</h3><ul><li><a href="AddEduForm.html">AddEduForm</a></li><li><a href="AddExpForm.html">AddExpForm</a></li><li><a href="businessProfile.html">businessProfile</a></li><li><a href="Contact.html">Contact</a></li><li><a href="CreateAddressForm.html">CreateAddressForm</a></li><li><a href="JobDescription.html">JobDescription</a></li><li><a href="ModalForm.html">ModalForm</a></li><li><a href="Profile.html">Profile</a></li><li><a href="UpdateAddressForm.html">UpdateAddressForm</a></li></ul><h3>Global</h3><ul><li><a href="global.html#AppliedJob">AppliedJob</a></li><li><a href="global.html#Education">Education</a></li><li><a href="global.html#Experiences">Experiences</a></li><li><a href="global.html#General">General</a></li></ul>
</nav>

<br class="clear">

<footer>
    Documentation generated by <a href="https://github.com/jsdoc/jsdoc">JSDoc 3.6.2</a> on Tue May 21 2019 01:26:30 GMT-0400 (EDT)
</footer>

<script> prettyPrint(); </script>
<script src="scripts/linenumber.js"> </script>
</body>
</html>