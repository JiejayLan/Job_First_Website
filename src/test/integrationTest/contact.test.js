import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Contact from '../../pages/contact';

describe('Contact page screenshot test', () => {
    test('Contact page should render correctly', () => {
        const renderer = new ReactShallowRenderer();
        renderer.render(<Contact />);
        expect(renderer.getRenderOutput()).toMatchSnapshot();
    });
});