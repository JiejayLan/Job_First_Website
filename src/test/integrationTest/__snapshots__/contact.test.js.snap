// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact page screenshot test Contact page should render correctly 1`] = `
<div
  align="center"
  className="mainDiv"
  style={
    Object {
      "backgroundImage": "url('https://images.pexels.com/photos/990818/pexels-photo-990818.jpeg?cs=srgb&dl=agenda-analysis-business-plan-990818.jpg&fm=jpg')",
      "backgroundSize": "cover",
    }
  }
>
  <br />
  <h1
    align="center"
    className="title"
    style={
      Object {
        "color": "white",
        "fontSize": "3em",
      }
    }
  >
    Contact Us
  </h1>
  <br />
  <br />
  <Column
    flexGrow={1}
  >
    <Row
      vertical="center"
    >
      <Column
        flexGrow={1}
        horizontal="center"
      >
        <Card
          size="default"
          style={
            Object {
              "backgroundColor": "#fcfcfc",
              "border": "2px solid black",
              "borderRadius": 15,
              "height": 210,
            }
          }
          title={
            <Icon
              style={
                Object {
                  "fontSize": 25,
                }
              }
              type="phone"
            />
          }
        >
          <br />
          <p
            className="description"
          >
            +1800JOBFIRST
          </p>
        </Card>
      </Column>
      <Column
        flexGrow={1}
        horizontal="center"
      >
        <Card
          size="default"
          style={
            Object {
              "backgroundColor": "#fcfcfc",
              "border": "2px solid black",
              "borderRadius": 20,
              "height": 210,
              "width": 250,
            }
          }
          title={
            <Icon
              style={
                Object {
                  "fontSize": 25,
                }
              }
              type="home"
            />
          }
        >
          <p
            className="description"
          >
            160 Convent Av,
            <br />
             New York, NY
            <br />
             10031
          </p>
        </Card>
      </Column>
    </Row>
  </Column>
</div>
`;
